tree/ntuple/v7/inc/ROOT/RNTupleUtil.hxx

```diff
@@ -328,6 +334,8 @@ RResult<void> EnsureValidNameForRNTuple(std::string_view name, std::string_view
 
 } // namespace Internal
 
+namespace Experimental {
```

Comment:
```suggestion
namespace Experimental {

namespace Internal {
```
isn't it? (to be consistent with previous version)

Reply:
The `EColumnType` typedef was not in `Internal`, it should be correct as-is